### PR TITLE
docs: Restyle readme

### PR DESCRIPTION
**Pull request by mtrezza**
Minor readme restyling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,11 @@ Replace `<VERSION>` with the version you want to install.
 ## Configure Parse Server
 
 ```js
-const PushAdapter = require('@parse/push-adapter').default;
+import { ParsePushAdapter } from '@parse/push-adapter';
```
Comment (dplewis) [inline README.md:38]: Is this the proper way to import?

File: README.md [hunk 2]
```diff
@@ -35,7 +35,11 @@ Replace `<VERSION>` with the version you want to install.
...
+
+// For CommonJS replace the import statemtent above with the following line:
+// const PushAdapter = require('@parse/push-adapter').default;
+
 const parseServerOptions = {
   push: {
     adapter: new PushAdapter({
```
Comment (dplewis) [inline README.md:45]: This doesn't match the import above

**Review by dplewis — COMMENTED** [review]
A few comments

File: README.md (hunk 2 above)
Comment (mtrezza) [reply README.md:45]: ES6 import imports the default export

Comment (mtrezza) [reply README.md:45]: I see what you mean, fixed in https://github.com/parse-community/parse-server-push-adapter/pull/249

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.3.0](https://github.com/parse-community/parse-server-push-adapter/releases/tag/6.3.0)

**Pull request merged**